classes/person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    true if is_of_age? || @parent_permission
+    false
+  end
+
+  private
+
+  def of_age?
+    true if @age >= 18
+    false
+  end
```

Comment:
- [ ] [_Optional_] Same here, the `true` and `false` statements can be completely removed